binding.gyp

```diff
@@ -35,8 +35,9 @@
         'GCC_ENABLE_CPP_EXCEPTIONS': 'NO',
         'MACOSX_DEPLOYMENT_TARGET': '10.7'
       },
-      'include_dirs': [
+      'include_dirs+': [
```

Comment:
Could/should this be done below in the `libsass_ext` condition? Not too familar with the gyp syntax though